azkaban-common/src/main/java/azkaban/AzkabanCommonModule.java

```diff
@@ -158,4 +163,10 @@ public FileSystem createHadoopFileSystem(final Configuration hadoopConf) {
   public QueryRunner createQueryRunner(final AzkabanDataSource dataSource) {
     return new QueryRunner(dataSource);
   }
+
+  @Provides
```

Comment:
Why not 
    bind(MetricRegistry.class).in(Scopes.SINGLETON);
it also seems to work.